Orange/widgets/data/oweditdomain.py

```diff
@@ -412,6 +449,7 @@ def __init__(self):
         self.editor_stack = QStackedWidget()
 
         self.editor_stack.addWidget(DiscreteVariableEditor())
+        self.editor_stack.addWidget(TimeVariableEditor())
```

Comment:
I would add this after ContinuousVariableEditor.

this way, the next diff should register as an addition of `elif var.is_time` as the `elif var.is_continuous` branch could remain unmodified